### PR TITLE
Update chain spec for flaming fir 7

### PR DESCRIPTION
**Pull request by ddorgan**
Reset flaming fir to master as of today.

### CODE REVIEW

**Comment by bkchr** [comment]
Can you please increment the `spec_version`?

File: bin/node/cli/src/chain_spec.rs [hunk 1]
```diff
@@ -139,7 +139,7 @@ fn staging_testnet_config_genesis() -> GenesisConfig {
 	// generated with secret: subkey inspect "$secret"/fir
 	let root_key: AccountId = hex![
 		// 5Ff3iXP75ruzroPWRP2FYBHWnmGGBSb63857BgnzCoXNxfPo
-		"9ee5e5bdc0ec239eb164f865ecc345ce4c88e76ee002e0f7e318097347471809"
+		"14201f475f63a18490b0dea51e87afba971d5944e75e78f3115a180cf9ec163a"
```
Comment (andresilva) [inline bin/node/cli/src/chain_spec.rs:142]: Did the secret change or is it being generated with a different scheme than `$secret/fir`? Could you update the address above? :)

Comment (ddorgan) [reply bin/node/cli/src/chain_spec.rs]: Switched back, thought I had lost the key. :)

**Comment by ddorgan** [comment]
Will reset flaming fir shortly again with these updates if there's no more feedback.

**Pull request merged**